rfc/0002-database_blocks.md

```diff
@@ -0,0 +1,85 @@
+# Name
+[name]: #name
+
+Database Blocks
+
+# Author
+[author]: #author
+
+Dragan Milic
+
+# Summary
+[summary]: #summary
+
+This is a description of how single memory mapped file is split into smaller units (blocks) that can be used to store data.
+
+# Motivation
+[motivation]: #motivation
+
```

Comment:
The Motivation section feels a bit rushed. It needs more context to build the big picture.

Reply:
you're right, fixed that now.